### PR TITLE
Fix ng13 esm example

### PR DESCRIPTION
**Pull request by johncrim**
## Summary

Fixes: #1222 

Updates the ng13 example so testing can be run using ES modules. This may not be the preferred fix, since per @ahnpnl it's not yet possible to use a setup-jest.mjs in the preset that is run automatically when ESM mode is enabled; but it does provide a working example for developers who want to use jest, ng13, and ESM now.

## Test plan

Running `yarn test-examples` succeeds. Since I don't know if this is in the CI, both of these work:

```
yarn
yarn test-examples
```

```
cd examples\example-app-v13
yarn
yarn test-esm # all tests pass
yarn test-esm-isolated # all tests pass
```

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by johncrim** [comment]
I also updated the docs. Lmk if you'd like me to remove the doc changes - it's certainly possible that this isn't ready for prime-time yet.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (ahnpnl) [inline .vscode/launch.json:1]: this file should be removed :)

File: examples/example-app-v13/setup-jest.ts [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+import 'zone.js/fesm2015/zone-testing-bundle.min.js';
 import './jest-global-mocks';
+
+import { getTestBed } from '@angular/core/testing';
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
+
+getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
+  teardown: {
+    destroyAfterEach: true,
```
Comment (ahnpnl) [inline examples/example-app-v13/setup-jest.ts:12]: this one is default set in Angular 13, we don't need to set here

File: package.json [hunk 3]
```diff
@@ -33,8 +33,8 @@
     "postinstall": "husky install",
     "lint": "eslint --ext .js,.ts .",
     "lint-fix": "eslint --fix --ext .js,.ts .",
-    "lint-prettier": "prettier '**/*.{yml,yaml,md}' --write",
-    "lint-prettier-ci": "prettier '**/*.{yml,yaml,md}' --check",
+    "lint-prettier": "prettier \"**/*.{yml,yaml,md}\" --write",
```
Comment (ahnpnl) [inline package.json:36]: I think these 2 should be reverted?

File: website/docs/guides/angular-13+.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ title: Angular >=13
```
Comment (ahnpnl) [inline website/docs/guides/angular-13+.md:5]: Would you please also apply the same change for https://github.com/thymikee/jest-preset-angular/blob/main/website/versioned_docs/version-11.x/guides/angular-13%2B.md

File: .vscode/launch.json (hunk 1 above)
Comment (johncrim) [reply .vscode/launch.json:1]: ok - apologies - I didn't intend to commit it.

File: examples/example-app-v13/setup-jest.ts (hunk 2 above)
Comment (johncrim) [reply examples/example-app-v13/setup-jest.ts:12]: Are you sure about this? My understanding is that this is set in new projects (in the karma setup file), and is recommended, but it's not a default. I'll step through to figure that out - and if you're right, I'll also remove from docs.

Comment (ahnpnl) [reply examples/example-app-v13/setup-jest.ts:12]: Ye it's a part of Angular 13 change, you can see it here https://github.com/ngrx/platform/issues/3243#issuecomment-973038378

File: package.json (hunk 3 above)
Comment (johncrim) [reply package.json:36]: These changes were necessary to run prettier on windows (with yarn 3). On the command line I can:

```
yarn prettier '**/*.{yml,yaml,md}' --write
```

But for some reason the commit hook always fails on this line with the previous code, but works with the change. I figure others could have the same problem, which is why I included this change. If you'd like it reverted, I'm happy to, but I will have to commit with `--no-verify`.

Comment (ahnpnl) [reply package.json:36]: hmm this is odd. If so, I think it's good to have.

File: examples/example-app-v13/setup-jest.ts (hunk 2 above)
Comment (johncrim) [reply examples/example-app-v13/setup-jest.ts:12]: You're right; I'll remove.

**Comment by johncrim** [comment]
@ahnpnl - I've addressed your feedback (thanks!). Lmk if you see anything else.

**Review by ahnpnl — APPROVED** [review]
Thanks!

**Pull request merged**